slitlessutils/core/modules/extract/single/single.py

```diff
@@ -57,7 +74,7 @@ class Single(Module):
     parent class.
 
     >>> import slitlessutils as su  # doctest: +SKIP
-    >>> ext = su.modules.Single('+1', mskorders=None, root=ROOT)  # doctest: +SKIP
+    >>> ext = su.modules.Single('+1', mskorders=None, outroot=ROOT)  # doctest: +SKIP
```

Comment:
Should this still be `root`, not `outroot`? I don't see `outroot` anywhere else here.